sdk/provisioning/Azure.Provisioning.Storage/tests/BasicStorageTests.cs

```diff
@@ -23,7 +23,7 @@ await test.Define(
             new StorageAccount("storage", StorageAccount.ResourceVersions.V2023_01_01)
             {
                 Kind = StorageKind.StorageV2,
-                Sku = new StorageSku { Name = StorageSkuName.StandardLrs },
+                Sku = { Name = StorageSkuName.StandardLrs },
```

Comment:
Now we'll lazily initialize models for you so you can just `storage.Sku.Name = ...` if you want without a `new StorageSku();` first.